script/Arcueid/Day 4/04_00_ARC04_10.txt

```diff
@@ -16,6 +16,7 @@ A cheerful sound rings out, terribly out-of-place.
 [sha:17e7cd0042013da2d82d7878318adf9d7b146276]{
 -- Page 10029, Offset 7781.
 -- 先ほどの箱のとなりにある、もう一つの箱が到着する。
+// cab? is this another term for an elevator?
```

Comment:
i think this was one we discussed a long time ago. cab is the term for the actual elevator doo-hicky while "elevator" is the whole system. I'm fine changing this to elevator

Reply:
I guess if its come up this much I prefer elevator.